spec/beta/rate_spec.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+describe EasyPost::Beta::Rate do
+  describe '.list' do
```

Comment:
The `describe` block is supposed to define a function or class being tested
```suggestion
  describe '.retrieve_stateless_rate' do
```